src/legacy/core_plugins/kibana/public/visualize/editor/editor.html

```diff
@@ -1,55 +1,60 @@
 <visualize-app class="app-container visEditor visEditor--{{ vis.type.name }}">
-  <!-- Local nav. -->
-  <kbn-top-nav name="visualize" config="topNavMenu">
-    <!-- Transcluded elements. -->
-    <div data-transclude-slots>
-      <!-- Search. -->
-      <div
-        data-transclude-slot="bottomRow"
-        ng-show="chrome.getVisible()"
-        class="fullWidth"
+  <!-- Linked search. -->
```

Comment:
For the linked search, I had given @timroes [a mock](https://github.com/elastic/kibana/issues/38272#issuecomment-509674634) to move this to where the current index-pattern is shown like so:

![image](https://user-images.githubusercontent.com/549577/61559750-69b6f800-aa38-11e9-9b23-9cd68fdd5734.png)

Reply:
Great to have this linked.